firedrake/ml/jax/fem_operator.py

```diff
@@ -92,7 +92,7 @@ def bwd(self, _, grad_output: "jax.Array") -> "jax.Array":
             adj_input = float(adj_input)
 
         # Compute adjoint model of `F`: delegated to pyadjoint.ReducedFunctional
-        adj_output = self.F.derivative(adj_input=adj_input)
+        adj_output = self.F.derivative(adj_input=adj_input, options={'riesz_representation': None})
```

Comment:
Is this equivalent to `'riesz_representation': 'l2'`?

Reply:
In terms of data, yes. I think PyTorch and JAX work with derivatives rather than gradients. So, returning a derivative (`Cofunction`) instead of converting it to a `Function` makes more sense to me.

Reply:
Okay, so this is different, as it keeps `Cofunctions` as `Cofunctions`.